TACC/Core-Portal
# task/FP-1307  Update ticket attachment design

## Overview: ##

## Related Jira tickets: ##

* [FP-1307](https://jira.tacc.utexas.edu/browse/FP-1307)

## Summary of Changes: ##
Update attachment design to match FP-403
## Testing Steps: ##
1. Open a ticket with an attachment and check if the new designed tickets applied.

## UI Photos:
New Design:
![Screen Shot 2021-11-09 at 12 47 38 PM](https://user-images.githubusercontent.com/34716439/140986117-2ff6c063-b71c-4e36-a0e7-8ad3047193b0.png)
Old Design:

![135889882-ca1f34b7-0cc9-46eb-a62e-d4ca2cfb94fc](https://user-images.githubusercontent.com/34716439/140986430-1f2f25e6-dc70-4348-acb7-537e1400fff7.png)

## Notes: ##
